### DISCUSSION TITLE
When SD3 training?

### DISCUSSION THREAD

**Discussion by QuantumState745837** [bmaltais/kohya_ss#2593] [General]
So I've seen some people trying to do Lora or Dreambooth training with SD3 already. 

https://huggingface.co/stabilityai/stable-diffusion-3-medium/discussions/98
https://github.com/huggingface/diffusers/blob/main/examples/dreambooth/README_sd3.md

Would SD3 be implemented soon?

**Comment by Vigilence** [comment]
Hi @bmaltais 

Any update on this? 3.5 SD3.5 Large is now out.
https://huggingface.co/stabilityai/stable-diffusion-3.5-large
